app/feature.go

```diff
@@ -0,0 +1,97 @@
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	ExportFeatureFlag = "enable-export"
+)
+
+func GetFeatureFlagConfigPath() string {
+	return ConfigDirFlag.Value + "/feature.json"
+}
+
+func GetFeatureFlagConfig(featureFlagConfigPath string) (map[string]bool, error) {
+	// create config file if not exist
+	if _, err := ioutil.ReadFile(featureFlagConfigPath); err != nil {
```

Comment:
we are using os.Stat to determine if files exist--I believe that is more standard?
```	tokenConfig := getTokenConfigPath(ctx)
	if _, err := os.Stat(tokenConfig); err != nil {
		// Skip if file does not exist
		if errors.Is(err, os.ErrNotExist) {
			return tokens, nil
		}
		return tokens, err
	}
```

Reply:
also should we just return early instead of creating the config? that way we can just create the config at creation time.